federation-2/router-bridge/js-src/plan_worker.ts

```diff
@@ -204,9 +211,16 @@ async function run() {
       }
     } catch (e) {
       logger.warn(`plan_worker: an unknown error occured ${e}\n`);
+      let unexpectedError = {
+        name: "unexpectedError",
+        message: `${e}`,
```

Comment:
This notation (``` `${e}` ```) will result in invoking `e.toString()` on the `e` object _if that method is defined_.  In **most** cases, that is defined as `self.message + "\n" + self.stack`, where [`stack` is the JS part of the stack](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/stack).

It's worth noting that the stack will be largely bundled symbols and relatively hard to interpret meaning and map it to code.  I expect it will be most valuable to us, rather than the client, whereas `message` might have meaning.

**Perhaps** to provide a nicer message, we might want to capture [`e.message` (in particular)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/message) here and have another property on the error object which captures the string serialized message that includes `stack`?  Perhaps even a third dedicated property for `stack`?